core/deployment/src/main/java/io/quarkus/deployment/steps/ClassTransformingBuildStep.java

```diff
@@ -104,7 +107,22 @@ TransformedClassesBuildItem handleClassTransformation(List<BytecodeTransformerBu
                                     visitor = i.apply(className, visitor);
                                 }
                                 cr.accept(visitor, 0);
-                                return new TransformedClassesBuildItem.TransformedClass(className, writer.toByteArray(),
+                                byte[] data = writer.toByteArray();
+                                if (BootstrapDebug.DEBUG_TRANSFORMED_CLASSES_DIR != null) {
+                                    File debugPath = new File(BootstrapDebug.DEBUG_TRANSFORMED_CLASSES_DIR);
+                                    if (!debugPath.exists()) {
+                                        debugPath.mkdir();
+                                    }
+                                    File classFile = new File(debugPath, className.replace(".", "/") + ".class");
```

Comment:
I always forget - does this scheme of using `/` character as a file path seperator in the `File` constructor work fine on Windows?

Reply:
yes